tools/src/main/python/dlpx/virtualization/_internal/plugin_importer.py

```diff
@@ -199,6 +199,26 @@ def _get_manifest(queue, src_dir, module, entry_point, plugin_type, validate):
         #
         return
 
+    manifest = get_manifest_helper(src_dir, module, entry_point,
```

Comment:
I know this is not the code you've changed in this review but maybe we can raname `_get_manifest` to `_import_module_and_get_manifest`. Then, we could rename `get_manifest_helper` to `get_manifest`.

Reply:
fixed